plugin/dns64/README.md

```diff
@@ -70,6 +72,15 @@ Enable translation even if an existing AAAA record is present.
 }
 ~~~
 
+Apply translation even to the requests which arrived over IPv4 network.
```

Comment:
Can we add a note that this will break `A` record lookup for users coming from dual-stack networks?

Reply:
added a warning, hope the wording is understandable 👍